rpcapd/daemon.c

```diff
@@ -2855,7 +2861,7 @@ void sleep_secs(int secs)
 {
 #ifdef _WIN32
```

Comment:
It's probably cleaner to wrap the entire routine in

    #ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION

        ...

    #endif

Reply:
ok